ci-operator/config/maistra/istio-operator/maistra-istio-operator-maistra-3.0.yaml

```diff
@@ -71,6 +74,7 @@ tests:
     dependencies:
       RELEASE_IMAGE_LATEST: release:latest
     env:
+      BUILD_WITH_CONTAINER: "0"
```

Comment:
Is this env working in multi-steps job ?
The make execution is in a remote pod and we trigger it from a shell
oc rsh -n "${MAISTRA_NAMESPACE}" "${MAISTRA_SC_POD}" \
          entrypoint \
          sh -c \